en/companion_computer/holybro_pixhawk_rpi_cm4_baseboard.md

```diff
@@ -0,0 +1,216 @@
+# Holybro Pixhawk RPi CM4 Baseboard
+
+# Overview
+
+
+<img src="../../assets/companion_computer/holybro_pixhawk_rpi_cm4_baseboard/HB_Pixhawk_CM4_Basebooard_Main_Photo.jpg" width="580px" title="holybro_pixhawk_rpi_cm4_baseboard" />
+
+The Holybro Pixhawk RPi CM4 Baseboard combine the Pixhawk FC module with the Raspberry Pi CM4 companion computer in on compact form factor with all the connections you need for development. \
+\
+It follows the Pixhawk Connector and Autopilot Bus Standard, allow easy swap of FC Module with any FC that follows the Pixhawk Bus Standard. The FC Module is internally connected to RPi CM4 through TELEM2, and can also be connected via ethernet with a external cable provided.  This baseboard is compatible with both Pixhawk 5X & 6X, and most other flight controller that follow the Pixhawk Autopilot Bus Standard.\
```

Comment:
@vincentpoont2 Is this true "It follows the Pixhawk Connector and Autopilot Bus Standard, allow easy swap of FC Module with any FC that follows the Pixhawk Bus Standard." ?

I mean the standard defines the connector but not its physical location compared to the form factor of your FC. So what you mean is that you can swap out Holybro FCs, and with some work you might connect other FCs - right?